application/src/main/java/org/opentripplanner/service/vehiclerental/model/VehicleRentalStation.java

```diff
@@ -250,7 +251,9 @@ public boolean canDropOffFormFactor(
     RentalFormFactor formFactor,
     boolean includeRealtimeAvailability
   ) {
-    if (returnPolicy.equals(ReturnPolicy.ANY_TYPE) && allowDropoffNow()) {
+    if (returnPolicy.equals(ANY_TYPE) && includeRealtimeAvailability) {
+      return allowDropoffNow();
+    } else if (returnPolicy.equals(ANY_TYPE)) {
       return true;
     } else {
```

Comment:
I think the logic should be:
if (returnPolicy.equals(ANY_TYPE)) {
    return includeRealtimeAvailability ? allowDropoffNow() : isAllowDropoff()
} else ...